tests/unit/spec/mozilla-smoothscroll.js

```diff
@@ -0,0 +1,84 @@
+/* For reference read the Jasmine and Sinon docs
+ * Jasmine docs: https://jasmine.github.io/2.3/introduction.html
+ * Sinon docs: http://sinonjs.org/docs/
+ */
+
+/* global describe, beforeEach, afterEach, it, expect, sinon, spyOn */
+
+describe('mozilla-smoothscroll.js', function () {
+
+    'use strict';
+
+    describe('Mozilla.smoothScroll scrolling', function () {
+
+        beforeEach(function () {
+            var scrollMarkup = [
+                '<div id="scrollMarkup">',
+                    '<nav id="nav">',
+                        '<a id="link1" href="#anchor1">Anchor 1</a>',
+                        '<a id="link2" href="#anchor2">Anchor 2</a>',
+                        '<a id="link3" href="#anchor3">Anchor 3</a>',
+                    '</nav>',
+                    '<div id="anchor1" style="height: 400px;"></div>',
+                    '<div id="anchor2" style="height: 400px;"></div>',
+                    '<div id="anchor3" style="height: 400px;"></div>',
+                '</div>'
+            ].join();
+
+            $(scrollMarkup).appendTo('body');
+        });
+
+        afterEach(function () {
+            $('#scrollMarkup').remove();
+        });
+
+        it('should use window.scrollTo when native smooth scrolling is supported', function () {
+            spyOn(window, 'scrollTo');
+
+            Mozilla.smoothScroll({
+                top: $('#anchor1').offset().top,
+                unitTest: 'native'
+            });
+
+            expect(window.scrollTo).toHaveBeenCalledWith({
+                behavior: 'smooth',
+                top: $('#anchor1').offset().top,
```

Comment:
Ah, no, not anymore. I originally was going to test that the window scrolled to the correct position, but it doesn't appear the window scrolls at all when using Karma (even though the calls to scroll are made). Will simplify.